src/ppom-controller.ts

```diff
@@ -115,7 +120,10 @@ const versionInfoFileHeaders = {
 
 export type UsePPOM = {
   type: `${typeof controllerName}:usePPOM`;
-  handler: (callback: (ppom: any) => Promise<any>) => Promise<any>;
+  handler: (
+    callback: (ppom: any) => Promise<any>,
+    networkClientId?: NetworkClientId,
+  ) => Promise<any>;
```

Comment:
We should be able to use `unknown` here without disruption, since `any` is assignable to `unknown` and `unknown` is assignable to `any`. 

```suggestion
    callback: (ppom: unknown) => Promise<unknown>,
    networkClientId?: NetworkClientId,
  ) => Promise<unknown>;
```